src/app.py

```diff
@@ -783,11 +821,40 @@ def app_manifest(app):
 
     manifest, extracted_app_folder = _extract_app(app)
 
-    shutil.rmtree(extracted_app_folder)
-
     raw_questions = manifest.get("install", {}).values()
     manifest["install"] = hydrate_questions_with_choices(raw_questions)
 
+    # Add a base64 image to be displayed in web-admin
+    if Moulinette.interface.type == "api":
+        import base64
+
+        manifest["image"] = None
+        screenshots_folder = os.path.join(extracted_app_folder, "doc", "screenshots")
+
+        if os.path.exists(screenshots_folder):
+            with os.scandir(screenshots_folder) as it:
+                for entry in it:
+                    ext = os.path.splitext(entry.name)[1].replace(".", "").lower()
+                    if entry.is_file() and ext in ("png", "jpg", "jpeg", "webp"):
```

Comment:
```suggestion
                    if entry.is_file() and ext in ("png", "jpg", "jpeg", "webp", "gif"):
```

There's also a bunch of apps shipping a .gif ... do you think those may work too on the webadmin ?